src/main/scala/com/cognite/sdk/scala/v1/propertyMap.scala

```diff
@@ -67,26 +76,29 @@ object PropertyMap {
 }
 
 final case class Node(
+    override val spaceExternalId: Option[String],
     override val externalId: String,
     properties: Option[Map[String, DataModelProperty[_]]] = None
 ) extends PropertyMap(
       {
-        val propsToAdd: Map[String, DataModelProperty[_]] =
+        val propsToAdd: Map[String, DataModelProperty[_]] = spaceExternalId
```

Comment:
Break it out to `val spaceProps = spaceExternalId...` and `val propsToAdd = spaceProps ++ Map("externalId" -> ...)`

Reply:
they remove spaceExternalId at this level from DMS api so I remove this prop as well 😅